generate-release-cycle.py

```diff
@@ -0,0 +1,136 @@
+"""
+Read in a JSON and generate two CSVs and a Mermaid file
+"""
+from __future__ import annotations
+
+import csv
+import datetime as dt
+import json
+
+MERMAID_HEADER = """
+gantt
+    dateFormat  YYYY-MM-DD
+    title       Python release cycle
+    axisFormat  %Y
+""".lstrip()
+
+MERMAID_SECTION = """
+    section Python {version}
+    {release_status}               :{mermaid_status}   python{version}, {release_date},{eol}
+"""  # noqa: E501
+
+MERMAID_STATUS_MAPPING = {
+    "feature": "",
+    "bugfix": "active,",
+    "security": "done,",
+    "end-of-life": "crit,",
+}
+
+CSV_HEADER = (
+    "Branch",
+    "Schedule",
+    "Status",
+    "First release",
+    "End of life",
+    "Release manager",
+)
+
+
+def pep(number: int) -> str:
+    """Format PEP number with Sphinx role"""
+    return f":pep:`{number}`"
+
+
+def csv_date(date_str: str, now_str: str) -> str:
+    """Format a date for CSV"""
+    if date_str > now_str:
+        # Future, add italics
+        return f"*{date_str}*"
+    return date_str
+
+
+def mermaid_date(date_str: str) -> str:
+    """Format a date for Mermaid"""
+    if len(date_str) == len("yyyy-mm"):
+        # Mermaid needs a full yyyy-mm-dd, so let's approximate
+        date_str = f"{date_str}-01"
+    return date_str
+
+
+class Versions:
+    """For converting JSON to CSV and Mermaid"""
+
+    def __init__(self) -> None:
+        with open("include/release-cycle.json") as f:
+            self.versions = json.load(f)
```

Comment:
```suggestion
        with open("include/release-cycle.json", encoding="UTF-8") as in_file:
            self.versions = json.load(in_file)
```

Add missing encoding and use a more descriptive filename